src/platform-includes/sourcemaps/primer/javascript.sveltekit.mdx

```diff
@@ -0,0 +1,3 @@
+`@sentry/sveltekit` will generate and upload source maps automatically, so that errors in Sentry will show you readable stacktraces.
```

Comment:
```suggestion
`@sentry/sveltekit` will generate and upload source maps automatically, so that errors in Sentry will contain readable stack traces.
```